### DISCUSSION TITLE
colorByCategories legend item

### DISCUSSION THREAD

**Discussion by jung-han** [nhn/tui.chart#706] [Ideas]
![image](https://user-images.githubusercontent.com/35371660/135093046-a0f3ccea-6054-47fc-814e-6a34a1576edb.png)

@jwlee1108 Thanks for making a great chart! 👍👍
I have one question. When using the `colorByCategories` option, it seems that legend items should be created for each column. Is there any reason why it doesn't work that way? If there is no special reason, I would be grateful if you thought about it once!
(I think it would be better not to show up)

In addition, it would be better if there is an example page on the api site. Thanks!

**Comment by jung-han** [comment]
And congratulations over 5000 stars! 😻

**Comment by jwlee1108** [comment]
@jung-han 

Hello!  Thank you for your idea.
The basic role of Legend to the display of the series. Also, the `colorByCategories` option could apply not only to column charts but also to column line charts. So I make a decision to legend's standard as a series. 

![image](https://user-images.githubusercontent.com/30035674/133181927-0a0b4b85-3ada-4cd5-9727-a4a9ce7d01ed.png)  

I'm going to test the multi-series in more detail to refine this feature and add an example to API Site.
